### ISSUE TITLE
capmonster.cloud can't bypass captcha

### ISSUE THREAD

**Issue by lukosevic** [V4NSH4J/discord-mass-DM-GO#514]
![Screenshot_83](https://user-images.githubusercontent.com/80982331/163460711-93c4fca4-2860-4c1a-b370-3bec8bb5d2e2.png)
![Screenshot_84](https://user-images.githubusercontent.com/80982331/163460721-27e66020-e8b6-47db-9786-7f5849cfc23e.png)

**Comment by V4NSH4J** [comment]
Hey, I fixed the error. Please re-download the file.

**Comment by lukosevic** [comment]
> Hey, I fixed the error. Please re-download the file.

thanks bro, it's working now. Moreover in what format should I use proxies in proxy.txt? IP:port or username:password@domain:port

**Comment by lukosevic** [comment]
and does it supposed to resolve the captcha every each DM sent?

**Comment by V4NSH4J** [comment]
> > Hey, I fixed the error. Please re-download the file.
> 
> thanks bro, it's working now. Moreover in what format should I use proxies in proxy.txt? IP:port or username:password@domain:port

Hey you should use username:password@ip:port or ip:port 
Either is fine, depending on how your proxies are authenticated.

**Comment by V4NSH4J** [comment]
> and does it supposed to resolve the captcha every each DM sent?

Yes, after a few DMs, every or every couple DMs require captcha.
